src/ActionsImporter/Commands/Common.cs

```diff
@@ -73,6 +79,15 @@ public static Command AppendTransformerOptions(this Command command)
         return command;
     }
 
+    public static Command AppendPrereleaseOption(this Command command)
+    {
+        ArgumentNullException.ThrowIfNull(command);
+
+        command.AddGlobalOption(Prerelease);
+
+        return command;
+    }
+
```

Comment:
This can probably go under `AppendGeneralOptions`, that way we don't have to change `Forecast.cs`